app/components/files/file.tsx

```diff
@@ -30,7 +30,8 @@ type FileProps = {
     onPress: (index: number) => void;
     publicLinkEnabled: boolean;
     channelName?: string;
-    onOptionsPress?: (index: number) => void;
+    onOptionsPress?: (fileInfo: FileInfo) => void;
```

Comment:
Just a suggestion: 
onOptionsPress is optional; that means that around line 180, you can check for it before calling the renderOptionsButton function.  That would prevent calling a function if onOptionsPress is undefined.

Reply:
Thanks.  I also converted renderOptionsButton from a function to a const after conditionally rendering the component